recipes/utils/ttysnoop.yaml

```diff
@@ -0,0 +1,46 @@
+inherit: [patch, make, install, cpackage]
+
+metaEnvironment:
+    PKG_VERSION: "0.12d"
+    PKG_VERSION_PATCH: "0.12d-6"
+
+depends:
+    - libs::libxcrypt-dev
+    - use: []
+      depends:
+          - libs::libxcrypt-tgt
+
+checkoutSCM:
+    - scm: url
+      url: http://deb.debian.org/debian/pool/main/t/ttysnoop/ttysnoop_${PKG_VERSION}.orig.tar.gz
```

Comment:
I think it is better to use this from `https://snapshot.debian.org/archive/debian/...`. We already had issues in the past because the "normal" mirror will only have the latest version. (see for example net::ifupdown)